package.json

```diff
@@ -76,7 +76,7 @@
     "@types/express": "4.17.17",
     "@types/node": "20.2.5",
     "@types/yargs": "17.0.24",
-    "@zeit/ncc": "0.22.3",
+    "@vercel/ncc": "0.36.1",
```

Comment:
I updated this package because @zeii/ncc is deprecated and did not work for `pnpm action`

```
> graphql-inspector-mono-repository@ action /Users/preston/projects/graphql-inspector
> ncc build packages/action/src/action.ts --out action --minify --transpile-only

ncc: Version 0.22.3
ncc: Compiling file index.js
ncc: Using typescript@5.1.3 (local user-provided)
Error: Module parse failed: Unexpected token (79:32)
File was processed with these loaders:
 * ./node_modules/.pnpm/@zeit+ncc@0.22.3/node_modules/@zeit/ncc/dist/ncc/loaders/empty-loader.js
 * ./node_modules/.pnpm/@zeit+ncc@0.22.3/node_modules/@zeit/ncc/dist/ncc/loaders/relocate-loader.js
 * ./node_modules/.pnpm/@zeit+ncc@0.22.3/node_modules/@zeit/ncc/dist/ncc/loaders/uncacheable.js
 * ./node_modules/.pnpm/@zeit+ncc@0.22.3/node_modules/@zeit/ncc/dist/ncc/loaders/ts-loader.js
 * ./node_modules/.pnpm/@zeit+ncc@0.22.3/node_modules/@zeit/ncc/dist/ncc/loaders/shebang-loader.js
You may need an additional loader to handle the result of these loaders.
|     }
|     let [schemaRef, schemaPath] = schemaPointer.split(':');
>     if (useMerge && pullRequest?.state === 'open') {
|         ref = `refs/pull/${pullRequest.number}/merge`;
|         workspace = undefined;
    at /Users/preston/projects/graphql-inspector/node_modules/.pnpm/@zeit+ncc@0.22.3/node_modules/@zeit/ncc/dist/ncc/index.js.cache.js:3:63173
    at eval (eval at create (/Users/preston/projects/graphql-inspector/node_modules/.pnpm/@zeit+ncc@0.22.3/node_modules/@zeit/ncc/dist/ncc/index.js.cache.js:1:275040), <anonymous>:13:1)
 ELIFECYCLE  Command failed with exit code 1.
```